addons/mail/static/src/core/web/store_service_patch.js

```diff
@@ -119,7 +119,7 @@ const StorePatch = {
     _onActivityBroadcastChannelMessage({ data }) {
         switch (data.type) {
             case "INSERT":
-                this["mail.activity"].insert(data.payload, { broadcast: false });
+                this.insert(data.payload, { broadcast: false });
```

Comment:
Should move out of activity

Reply:
The feature to broadcast data to other tab is currently activity specific, and non-official to store.
I'd say this out-of-scope of this PR, also because I'm pretty sure we don't want to broadcast the data to all tabs for all records at all time like activity: some fields like `Thread.messages` are very tab-dependent, other tabs can mess up the loading of messages.